vuepress/docs/v6.3.2/tutorials/samples/hub.md

```diff
@@ -51,33 +51,33 @@ TODO: screenshot of the 6.3.2 AppBuilder with both bundles available but not ins
 (Entando 6.3.2) There is a cache issue when deploying the Hub bundles where not all widgets or MFEs initially appear on some pages, particularly the Dashboard page. Restarting the quickstart-server pod (which holds the Entando App Engine) will clear the cache. This is only necessary on the initial install.
 :::
 
-5. The Hub can be accessed from the App Builder by navigating to `Pages → Management`, finding `xxxx` in the page tree, and clicking `View Published Page` from its actions.
+5. The Hub can be accessed from the App Builder by navigating to `Pages → Management`, finding `Hub Test` **change** in the page tree, and clicking `View Published Page` from its actions.
 
 ## Configuration
 
 ::: tip
-(New with Entando 7.0.0) To point the Entando App Builder in another Entando Application to any Hub instance, configure the App Builder with the URL for the Hub API: http://YOUR-BASE-URL/entando-hub-api/appbuilder/api.  For example, the public Entando Hub has this API: <http://hubdev.okd-entando.org/entando-hub-api/appbuilder/api>. **change**
+(New with Entando 7.0.0) To point the Entando App Builder in another Entando Application to any Hub instance, configure the App Builder using the endpoint `BASEURL/entando-hub-api/appbuilder/api`, where the BASEURL is the URL for the Entando Application.
```

Comment:
Is the important feature the availability of any Hub? IF that's the case, I'd unbury that from the middle of the sentence to the front, to lead with that ...something like  "Any Hub instance can be installed... "

Reply:
@jyunmitch hmm i don't think this pertains to hub installation; more like finding one instance's hub from another. rewording to lead with the hub instance while still remaining concise preserving the point is a little tricky... "Any Hub instance can be accessed from the Entando App Builder of another Entando Application. Configure the App Builder to access the desired Hub instance via the endpoint `BASEURL/entando-hub-api/appbuilder/api`, where the BASEURL is the URL for the Entando Application." ??? it seems like two sentences to break this up is a good move.